src/foam/nanos/crunch/ui/CapabilityWizardlet.js

```diff
@@ -25,7 +25,7 @@ foam.CLASS({
     {
       name: 'id',
       expression: function (capability) {
-        return 'capability,' + capability.id;
+        return capability ? 'capability,' + capability.id : 'capability undefined';
```

Comment:
"why do we add the 'capability,' prefix? Could we just use the capability.id?
Could this just be?
return capability ? capability.id : '';  ?

Reply:
So I believe the reason this was set to 'capability' prefix was cause the wizardlet is itself an object and also stores the ucj (as a property) <- or use too ... now wizardlet stores UCJWAO object as property.

Therefore the pt was to specify the id.

However Kgr brings up a good pt. 
Either this uses its own id or it should just return cap.id and it will be known that this is its convention.
I vote for removing 'capability' prefix.